src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml

```diff
@@ -26,6 +26,35 @@
             EmptyValue="Collapsed"
             NotEmptyValue="Visible" />
 
+        <StackLayout
+            x:Name="HorizontalStackLayout"
+            Orientation="Horizontal"
+            Spacing="8" />
+
+        <DataTemplate x:Key="TagTemplate" x:DataType="viewmodels:TagViewModel">
+            <!--  TODO: Actually colorize the tags again -->
+            <Border
```

Comment:
Yeah, we can use a converter or x:Bind function to take the `Color` and convert it to a brush (that could also do light/dark management of twisting the color or aligning them to a more fixed palette, etc...) - especially to balance with the text color, auto-accessibility built-in?

I'd imagine we'd want some sort of cache for the `SolidColorBrush`. I'm surprised we don't have anything in the Toolkit already. @Sergio0694 is this anything the store does we could up-lift to the toolkit? Or @niels9001 I suppose we could reach out to any other 1P app teams too.